src/components/tile.js

```diff
@@ -0,0 +1,40 @@
+((() => {
+  const html = `
+    <div class="tile">
+      <app :tiles="tiles" :matchingOptions="matchingOptions"></app>
+    </div>
+  `
+
+  Vue.component("tile", {
+    template: html,
+
+    data() {
+      return {
+        tiles: [],
```

Comment:
Not sure if this will work - but I was thinking of adding 

`numberOfMatches: 2` - this can change depending on user input/ level/ etc.

Reply:
hmm. I think we are trying to accomplish the same thing but in a different way. I am thinking instead of hard coding `{name: "C", pairs: 2},` 2 would be `numberOfMatches` variable which we would get from the user. That way we know that there needs to be three cards of each on the board. This will also let us know how many times we need to loop in the `populateBoard` function. We can basically just take the length of `matchingOptions` and multiply it by `numberOfMatches`